files/zh-cn/web/html/element/input/image/index.md

```diff
@@ -7,83 +7,323 @@ slug: Web/HTML/Element/Input/image
+- `post`
+  - : The form's data is included in the body of the request that is sent to the URL given by the `formaction` or {{htmlattrxref("action", "form")}} attribute using an HTTP {{HTTPMethod("post")}} request. This method supports complex data and file attachments.
+- `dialog`
+  - : This method is used to indicate that the button closes the dialog with which the input is associated, and does not transmit the form data at all.
+
+该属性也在 [`<input type="submit">`](/zh-CN/docs/Web/HTML/Element/input/submit) 和 {{HTMLElement("button")}} 元素上可用。
+
+### formnovalidate
+
+一个布尔值，如果存在，则指定在提交给服务器之前不应对表单进行验证。这将会覆盖元素所属表单上的 {{htmlattrxref("novalidate", "form")}} 属性的值。
+
+该属性也在 [`<input type="submit">`](/zh-CN/docs/Web/HTML/Element/input/submit) 和 {{HTMLElement("button")}} 元素上可用。
+
+### formtarget
+
+一个字符串，指定一个名字或关键词来表示在提交表单后在何处显示响应数据。该字符串必须是一个**浏览上下文**的名字（即一个标签页、窗口，或 {{HTMLElement("iframe")}}）。在此处指定的值将会覆盖 {{HTMLElement("form")}} 上拥有该输入的 {{htmlattrxref("target", "form")}} 属性所给出的任何目标。
+
+除了标签页、窗口或内联框架的实际名字，还有部分特殊关键词可用：
+
+- `_self`
+  - : 将响应加载到与包含表单的浏览上下文相同的浏览上下文中。这将用接收到的数据替换当前文档。如果未指定，这将是默认值。
+- `_blank`
+  - : 将响应加载到新的未命名浏览上下文中。这通常是一个与当前文档相同的窗口中的新选项卡，但是可能会因为 {{Glossary("user agent")}} 的配置而有所不同。
+- `_parent`
+  - : 将响应加载到当前浏览上下文的父浏览上下文中。若不存在父浏览上下文，该选项的行为将与 `_self` 相同。
+- `_top`
+  - : 将响应加载到顶级浏览上下文中；它是当前浏览上下文的最顶层祖先。如果当前浏览上下文为最顶层上下文，该选项的行为将与 `_self` 相同。
+
+该属性也在 [`<input type="submit">`](/zh-CN/docs/Web/HTML/Element/input/submit) 和 {{HTMLElement("button")}} 元素上可用。
+
+### height
+
+一个表示绘制 `src` 属性指定图像的高度的数字，以 CSS 像素为单位。
+
+### src
+
+一个字符串，指定将要在提交按钮上显示的图像的 URL。当用户与该图像交互时，输入将会以和其他按钮输入相同的方式被处理。
+
+### width
+
+一个表示绘制图像宽度的数字，以 CSS 像素为单位。
+
+## 被弃用的属性
+
+下列属性是由 HTML 4 为 `image` 输入定义的，但未被所有浏览器实现，并且已被弃用。
+
+### usemap
+
+如果 `usemap` 被指定，它必须是一个图像映射元素的名字。{{HTMLElement("map")}} 用于定义一个和图像一同使用的图像映射。该用法已被弃用；当你需要使用图像映射时，你应该切换到使用 {{HTMLElement("img")}} 元素。
+
+## 使用图像输入
+
+The `<input type="image">` element is a [replaced element](/en-US/docs/Web/CSS/Replaced_element) (an element whose content isn't generated or directly managed by the CSS layer), behaving in much the same way as a regular {{htmlelement("img")}} element, but with the capabilities of a [submit button](/en-US/docs/Web/HTML/Element/input/submit).
+
+### 基本的图像输入特性
+
+Let's look at a basic example that includes all the essential features you'd need to use (These work exactly the same as they do on the `<img>` element.):
+
+```html
+<input
+  id="image"
+  type="image"
+  width="100"
+  height="30"
+  alt="Login"
+  src="https://raw.githubusercontent.com/mdn/learning-area/master/html/forms/image-type-example/login.png" />
+```
+
+{{ EmbedLiveSample('Essential_image_input_features', 600, 50) }}
+
+- The [`src`](/en-US/docs/Web/HTML/Element/input#src) attribute is used to specify the path to the image you want to display in the button.
```

Comment:
Please update the url and let this link to `zh-CN` document. And we do not really like to add more english content in zh-cn documents. If you can, please translate all of the newly added content (or you can remove them for now).